pkgs/development/interpreters/dart/default.nix

```diff
@@ -25,6 +25,14 @@ let
       url = "${stable}/${version}/sdk/dartsdk-linux-ia32-release.zip";
       sha256 = "d67b8f8f9186e7d460320e6bce25ab343c014b6af4b2f61369ee83755d4da528";
     };
+    "2.0.0-x86_64-linux" = fetchurl {
+      url = "{$stable}/${version}/sdk/dartsdk-linux-x64-release.zip";
+      sha256 = "4014a1e8755d2d32cc1573b731a4a53acdf6dfca3e26ee437f63fe768501d336";
+    };
+    "2.0.0-i686-linux" = fetchurl {
+      url = "{$stable}/${version}/sdk/dartsdk-linux-ia32-release.zip";
```

Comment:
same issue as before.

Reply:
sorry for causing so many problems with this, it's been a little hectic not being able to fix things because my internet is spotty right now

Reply:
Thais all right. just take your time.